docs/_photoniq/fingerprinting/index.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 1
+title: Digital Fingerprinting
+---
+
+PhotonIQ Digital Fingerprinting is a managed service that generates persistent browser fingerprints and visitor IDs for your website users.
```

Comment:
```suggestion
PhotonIQ Digital Fingerprinting is a managed service that generates persistent digital signatures and visitor IDs for your website users.
```

Reply:
Question around  "website".  Will that always be the case?  If you are using a phone app, would that still apply here?  We initially are not targeting phone apps, but wondering if we should word this in such a way so that people don't automatically discount us because we only support "website" users.